ui/src/file_upload.tsx

```diff
@@ -173,11 +186,11 @@ export const
         }
         else {
           setFiles(fileArr)
+          setFileNames(fileArr.map(({ name }) => name).join(', '))
```

Comment:
Why did this move?

Reply:
Hm.. not sure what you mean, but this change introduces buggy behavior:


https://user-images.githubusercontent.com/64769322/177504235-608d7b4f-8b2b-4617-9e36-eabe7ae31f0a.mov